.travis.yml

```diff
@@ -17,7 +18,7 @@ notifications:
 script:
 - bundle exec rake default
 - gem build sensu-plugins-google-spreadsheet.gemspec
-- gem install sensu-plugins-google-spreadsheet-*.gem
+- gem install sensu-plugins-google-spreadsheets-*.gem
```

Comment:
I see: https://github.com/sensu-plugins/sensu-plugins-google-spreadsheet/blob/master/sensu-plugins-google-spreadsheet.gemspec#L29 I am gonna open an issue and see if we want to either rename the gem or the repo, I think its super confusing when these are not the same...